### PR TITLE
add travis build status to README.md

### PR DESCRIPTION
**Pull request by liangchenye**
Fixes:#54.
Signed-off-by: Liang Chenye <liangchenye@huawei.com>

### CODE REVIEW

**Comment by jodh-intel** [comment]
lgtm

(Looks like the Travis build is failing on master due to timeouts again @jcvenegas)

@klynnrif - could you take a look please?

**Comment by liangchenye** [comment]
@jodh-intel  it is still caused by EulerOS .
I'm looking into it.

**Comment by jodh-intel** [comment]
Thanks @liangchenye.

File: tests/image_creation.bats [hunk 1]
```diff
@@ -45,5 +45,9 @@ function build_image()
 }
 
 @test "Can create euleros image" {
+	if [ "$TRAVIS" = true ]
```
Comment (jodh-intel) [inline tests/image_creation.bats]: Hi @liangchenye - we might want to do this at some pont. However, this PR should just be for updating the README imho.

**Comment by liangchenye** [comment]
The travis said 'Operation too slow. Less than 1000 bytes/sec transferred the last 30 seconds' in EulerOS build. I tests the [community repo](https://www.isula.org/repo), it has 'https' support which is better, but I'm not sure the network is stable. 

I added a new commit (same as before) to skip the euleros build.
I'll test the euleros build in my environment once the building scripts changes or the euler repo updates.

File: tests/image_creation.bats (hunk 1 above)
Comment (liangchenye) [reply tests/image_creation.bats]: I'll revert it and add a new PR..

**Comment by liangchenye** [comment]
submit a new PR #56, sorry for the building issue.
PTAL

**Comment by jcvenegas** [comment]
lgtm

**Pull request merged**